woof-code/3builddistro

```diff
@@ -897,6 +897,7 @@ fi
 if [ "$FRUGALIFY" ]; then
 	echo "PUPMODE='6'" > rootfs-complete/etc/rc.d/PUPSTATE
 	echo "PUNIONFS='overlay'" >> rootfs-complete/etc/rc.d/PUPSTATE
+	echo "PUP_HOME=''" >> rootfs-complete/etc/rc.d/PUPSTATE
```

Comment:
If/when frlugalify runs with aufs instead of layerfs sfs_load may have problem with this